snow/engine/snowman/transitive_test.go

```diff
@@ -60,10 +64,21 @@ func BuildChain(root *snowman.TestBlock, length int) []*snowman.TestBlock {
 	return chain
 }
 
-func setup(t *testing.T) (ids.NodeID, validators.Manager, *common.SenderTest, *block.TestVM, *Transitive) {
-	require := require.New(t)
+func MakeGetBlockF(blks ...[]*snowman.TestBlock) func(context.Context, ids.ID) (snowman.Block, error) {
```

Comment:
Should this live in `snowtest`?

Reply:
That would cause a circular import... But I think it would make sense to move `*snowman.TestBlock` into a test package and then have these utils live there.

Do you think we should do that change in this PR or a followup?

Reply:
Can be a followup... don't see a point blocking on this